cip-drep-metadata/README.md

```diff
@@ -0,0 +1,231 @@
+
+`motivation`
+- Optional
+- A freeform text field with a maximum of 1000 characters
+- A short description of why they want to be a DRep, what personal and professional experiences have they had that have driven them to 
+
+`qualifications`
+- Optional
+- A freeform text field with a maximum of 1000 characters 
+- A space to list the qualifications that the subject of this metadata has that are relevant to being a DRep
+
+`identity`
+- Compulsory
+- A link to a social media profile where the person must reference their DRep ID in their profile.
+- This will be used by people reviewing this DRep to prove and verify that the person described in the metadata is the same as the person who set up the social media profile.
+- Tooling providers SHOULD warn people that none of the information is verified by the tool and they should DYOR
+- Tooling providers making metadata SHOULD provide some information about why this is important.
+
+PKI cryptographic solutions were also considered in the place of the current solution however they were generally considered too complex to implement for minimum viable tooling. There were also other issues with cryptographic forms of identification for MVPs:
+1. solutions that involve a public/private key setup still require people verifying the identity of a DRep to know what the authentic pulic key is.
+2. specifying the use of a verification service such as [Keybase](https://keybase.io/) would lead to centralisation and therefore reliance on a 3rd party for DReps.
+
+`links`
+- Optional
+- A list of urls to the social media/ websites associated with the DRep
+- It is up to the client to check where these links go
+- Warning about linking to external links
+
+`donotlist`
+- Optional
+- If not included then the value is assumed to be false
+- A boolean expression
+- A true value means that the DRep does not want to show up in tooling that displays DReps. 
+-- I.e. they do not want to appear in GovTool’s DRep Explorer feature
+
+
+### Application
+DRep metadata must include all compulsory fields to be considered CIP-???? compliant. As this is an extension to CIP-100, all CIP-100 fields can be included within CIP-???? compliant metadata.
+
+### Test Vector
+See test-vector.md for examples.
+
+### Versioning
+This proposal should not be versioned, to update this standard a new CIP should be proposed. Although through the JSON-LD mechanism further CIPs can add to the common governance metadata vocabulary,
+
+## Rationale: how does this CIP achieve its goals?
+We intentionally have kept this proposal brief and uncomplicated. This was to reduce the time to develop and deploy this standard. This way we enable tooling which depends on this standard to start development. The fields which have been chosen for this standard are heavily inspired by those that we are seeking to introduce for GovTool. We did this because GovTool will likely be the first technical implementation of this standard. 
+
+## Open Questions
```

Comment:
```suggestion
### Open Questions
```
This is only a major section for CPSs so I'd suggest demoting it to a sub-section of _Rationale_ since these questions all appear related to the approach you've taken or are considering taking.